pom.xml

```diff
@@ -127,6 +127,38 @@
                             <failOnUnableToExtractRepoInfo>false</failOnUnableToExtractRepoInfo>
                         </configuration>
                     </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-enforcer-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>enforce-java</id>
+                                <goals>
+                                    <goal>enforce</goal>
+                                </goals>
+                                <configuration>
+                                    <rules>
+                                        <requireJavaVersion>
+                                            <message>
+                                                🚫 Java version error: This project requires Java ${java.version} or higher.
+
+                                                Your current Java version is ${java.version.detected}
+                                                To fix this:
+                                                1. Download and install Java 17 or higher from your preferred vendor
```

Comment:
We could use hère the java.version property

Reply:
sure